src/Data/Row/Variants.hs

```diff
@@ -549,7 +549,7 @@ eraseZipSingle f x y = getConst $ metamorph @_ @fs @c @Either
             (Left us, Left vs) -> Left (Pair (VApS us) (VApS vs))
             _                  -> Right $ Const Nothing
 
-        doCons :: forall l (τ :: * -> *) ρ
+        doCons :: forall l τ ρ
```

Comment:
GHC doesn't seem to like this so much, and I'm not quite sure why.  Did you get it to type check?  Is it something about a change in a newer version of GHC?  This library uses many "new" features, but I don't really want it to be so bleeding edge that even GHC from a year ago can't compile it.